Core/include/Acts/Utilities/detail/periodic.hpp

```diff
@@ -15,7 +15,7 @@ namespace detail {
 
 /// Wrap a periodic value back into the nominal range.
 template <typename T>
-inline T wrap_periodic(T value, T start, T range) {
+constexpr inline T wrap_periodic(T value, T start, T range) {
   using std::floor;
```

Comment:
`std::floor` etc. will only get `constexpr` in C++23... So this won't work in `constexpr` functions anyways I think... But if it compiles its probably fine...

Reply:
interesting - this should then fail to compile no? I will revert anyways tho